src/content/docs/logs/ui-data/lookup-tables-ui.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: "Upload CSV-format lookup tables"
+metaDescription: 'In the New Relic logs UI, upload CSV-format lookup tables and use that data in combination with other New Relic data.'
+---
+
+import logsLookupTableUi from 'images/logs_screenshot-crop_lookup-table-ui.webp'
+
+Our **lookups** feature lets you enrich your log data, and other New Relic-stored data, with data about your business that you define in a CSV file. 
+
+## Why use lookups? [#overview]
+
+You can use lookups to make your telemetry data easier to understand, group data in custom ways, and include more information that wasn't available at the time for previously reported data. From the logs UI, you can upload a CSV file and use the `lookup()` function to include the lookup data in a NRQL query. 
+
+For reasons to use lookup tables, and how to use that data in queries, see [Query lookups](/docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/lookups).
```

Comment:
Should the link read "Use lookup table data in queries" instead of "Query lookups", since that's the actual title of the tutorial?